lib/view/json-view.js

```diff
@@ -13,7 +13,7 @@ function toString(jsonObject) {
     try {
         // handle issue when Error object serialized to {}
         if (jsonObject instanceof Error) {
-            jsonObject = { message: jsonObject.message, stack: jsonObject.stack };
+            jsonObject = { ...jsonObject, message: jsonObject.message, stack: jsonObject.stack };
```

Comment:
maybe better to wrap `...jsonObject` into a new property called `detail` and put it at the end. ```{ message, stack, detail }```

Reply:
thank you. updated.